docs/edgeql/ddl/links.rst

```diff
@@ -153,14 +153,16 @@ Define a new link ``interests`` on the ``User`` object type:
         CREATE MULTI LINK friends -> User
     };
 
-Define a new link ``friends_in_same_town`` as a computable on the
-``User`` object type:
+Define a new link ``special_group`` as a computable on the ``User``
+object type, which contains all the friends from the same town:
 
 .. code-block:: edgeql
 
     ALTER TYPE User {
-        CREATE LINK friends_in_same_town := (
-            SELECT __source__.friends FILTER .town = __source__.town)
+        CREATE LINK special_group := (
+            SELECT __source__.friends
+            FILTER .town = __source__.town
+        )
```

Comment:
Add a link to the "Computables" section, which explains what `__source__` is etc.

Reply:
I added links to computables from the links and properties examples.